SetSingleScreenVideoMode.brs

```diff
@@ -3,16 +3,25 @@
 ' Plugin return type - As String
 ' See BrightSign docs for a list of supported video modes - https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370676833/Supported+Video+Modes
 
+' Note: This plugin assumes it is executed on BrightSign Model with a single HDMI output. 
+' This will NOT work on a BrightSign Model with multiple HDMI outputs. 
+
 Function setvideomode(videoModeInputs, bsp) As String
```

Comment:
May be a naive question, but how is the input "videoModeInputs" being used here?

Reply:
I too have wondered this. The autorun passes this in as part of the Eval() operation. I believe what had happened is this bit had not been cleaned up when the initialization of `videoModeInputs` had been removed. This _may_ be an instance of leftover code.